data/dirt belt jobs.txt

```diff
@@ -0,0 +1,279 @@
+	on complete
+		payment 10000 160
+		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>. A group of <planet> citizens thank you and bid you farewell."
+
+
+mission "To Remembrance Day celebration [0]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Bring <fare> to <destination> by <date> for the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	to offer
+		random < 60
+		or
+			and
+				month == 12
+				day >= 21
+			month == 1
+			and
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment 1000 180
+		dialog "You wish your <passengers> the best of luck on <planet>. The children run off your ship in green face paint, and the parents pay you <payment>."
+
+
+mission "To Remembrance Day celebration [1]"
+	name `Remembrance Day celebration`
+	job
+	repeat
+	description `Deliver <cargo> to <destination> by <date> in preparation for the Remembrance Day celebration. Payment is <payment>.`
+	cargo random 5 2 .1
+	to offer
+		random < 60
+		or
+			and
+				month == 12
+				day >= 21
+			month == 1
+			and
+				month == 2
+				day < 19
+	to fail
+		month == 2
+		day == 19
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment 1000 180
+		dialog `Spaceport workers already festively dressed for Remembrance Day remove the cargo of <commodity> from your ship and hand you your payment of <payment>.`
+
+
+mission "From Remembrance Day celebration [0]"
+	name `Return from Remembrance Day`
+	job
+	repeat
+	description `Bring <fare> home to <destination> after the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .5
+	to offer
+		random < 60
+			and
+				month == 2
+				day > 19
+			month == 3
+			and
+				month == 4
+				day < 20
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment
+		payment 1000
+		dialog `Your passengers thank you for returning them to <planet>, and you collect your payment of <payment>.`
+
+
+mission "From Remembrance Day celebration [1]"
+	name `Return from Remembrance Day`
+	job
+	repeat
+	description `Bring <fare> home to <destination> after the Remembrance Day celebration. Payment is <payment>.`
+	passengers 4 4 .3
+	to offer
+		random < 60
+			and
+				month == 2
+				day > 19
+			month == 3
+			and
+				month == 4
+				day < 20
+	source
+		attributes "dirt belt"
+	destination
+		distance 3 15
+		attributes "dirt belt"
+	on complete
+		payment
+		payment 1000
+		dialog `Your passengers thank you for returning them to <planet>, and you collect your payment of <payment>.`
+
+
+mission "Food Convoy [0]"
+	name `Food convoy to <planet>`
+	job
+	repeat
+	description `The captain of the <npc> will pay you <payment> to escort his convoy of ships carrying food to <destination> by <date>.`
```

Comment:
It's not necessarily something I agree with, but I believe "his" should be changed to "their", for the sake of gender ambiguity.

Reply:
Yes, and in fact it should be rewritten to make the gender ambiguity not be awkward. For example:

`The captain of the <npc>, leading a convoy of ships carrying food, will pay you <payment> to escort them to <destination> by <date>.`

Or something like that.